third_party/objective_c/Cronet/BUILD

```diff
@@ -0,0 +1,27 @@
+# gRPC Bazel BUILD file.
+#
+# Copyright 2019 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+licenses(["notice"])  # Apache v2
+
+package(default_visibility = ["//:__subpackages__"])
+
+# C Header files used by GRPC C Transport.
+cc_library(
+    name = "cronet_c_for_grpc",
```

Comment:
Name it `Cronet` to match the internal version?

Reply:
It's the same name of internal version.

Reply:
Oh I see.